apps/builder/app/shared/router-utils/path-utils.ts

```diff
@@ -73,6 +73,8 @@ export const builderDomainsPath = (method: string) => {
   }`;
 };
 
+export const projectPath = (method: string) => `/project/${method}`;
```

Comment:
doesn't constrain anything, maybe just write path where it is used?

Reply:
we have all paths in this file, is spreading them across the codebase a good idea?

Reply:
What the different slightly refined path would make? "method" would spread the same way

Reply:
yes, its just about keeping all paths in one place and if needed extend from here in case we need a change